src/System.Globalization/tests/CultureInfo/CultureInfoAsync.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class CultureInfoAsync
+    {        
+        [Fact]
+        public void TestCurrentCulturesAsync()
```

Comment:
In addition to validating Task.Run, shouldn't we validate other forms of async operations? e.g. that the culture flows across awaits?